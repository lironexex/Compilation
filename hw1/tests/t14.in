// illegal strings 5 - unclosed string
"this string is closed"
"this string is not closed