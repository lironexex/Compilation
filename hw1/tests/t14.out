1 COMMENT 1
2 STRING this string is closed
Error unclosed string
